### ISSUE TITLE
Accessability - HTML errors

### ISSUE THREAD

**Issue by MalcolmShazell** [epimorphics/ukhpi#159]
It found a new one <sigh>
    Error: A document must not include more than one meta element with a charset attribute.

    From line 8, column 1; to line 8, column 22

    x</title>↩<meta charset='utf-8'>↩<scri

**Comment by ijdickinson** [comment]
Should be fixed now

**Comment by MalcolmShazell** [comment]
confirm fixed
